### PR TITLE
[CDAP-8366] Adds Resource center across UI

### PR DESCRIPTION
**Pull request by ajainarayanan**
[CDAP-8366](https://issues.cask.co/browse/CDAP-836)
- Adds ResourceCenterButton across different views
- Fixes on hover style for links in Header component

[CDAP-8101](https://issues.cask.co/browse/CDAP-8101)
  - Adds minimalist breadcrumb to detailed views.

[CDAP-8114](https://issues.cask.co/browse/CDAP-8114) 
  - Fixes Filter & Sort dropdowns in home page subnavbar to work in firefox

[CDAP-8420](https://issues.cask.co/browse/CDAP-8420) 
  - Fixes Resource Center title to Add Entity

[CDAP-8364](https://issues.cask.co/browse/CDAP-8364)
  - Fixes routes under metadata

### Note:
We still need to add the resource center button on tracker. Talking to Lea about the same.

### CODE REVIEW

**Comment by bdmogal** [comment]
qq: There's no text calling this Resource Center, right?

**Comment by ajainarayanan** [comment]
@bdmogal Yes. That is correct. Since I was used to calling it Resource Center I have used it in title. Its (+) button throughout

File: cdap-ui/app/cdap/components/Header/MetadataDropdown/index.js [hunk 1]
```diff
@@ -87,23 +87,32 @@ export default class MetadataDropdown extends Component {
...
+              href={dictionaryHomeUrl}>
               {T.translate('features.Navbar.Metadata.dictionaryLabel')}
             </a>
           </DropdownItem>
           <DropdownItem tag="ul" divider />
           <DropdownItem tag="li">
-            <a href={integrationsHomeUrl}>
+            <a
+              className={classnames({'active': integrationsHomeUrl === location.href})}
+              href={integrationsHomeUrl}>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Header/MetadataDropdown/index.js]: nit: `>` on newline

**Comment by elfenheart** [comment]
screen shot 2017-02-09 at 6 45 18 pm

The circle looks oval in Pipeline view

**Comment by elfenheart** [comment]
screen shot 2017-02-09 at 6 50 14 pm

This is for Dataset, but the breadcrumb is saying `Stream`

**Comment by elfenheart** [comment]
screen shot 2017-02-10 at 4 59 06 pm


Push the tags container down a little more?

Otherwise 👍  LGTM!!

**Pull request merged**